src/apps/api/views/competitions.py

```diff
@@ -72,6 +72,26 @@ def toggle_publish(self, request, pk):
         competition.save()
         return Response({"published": competition.published})
 
+    @action(detail=True, methods=('GET',))
+    def get_files(self, request, pk):
+        competition = Competition.objects.select_related(
```

Comment:
Hmm, this is OK but I wonder if this is nicer?

```python
qs_helper = Competition.objects.select_related('created_by').prefetch_related('dumps__dataset')
competition = qs_helper.get(id=pk)
```

Reply:
Can go either way? I think mine is more straightforward? but maybe that's just cause I wrote it ;) haha

Reply:
:nudges you towards my boring style: